DBusInterface.md

```diff
@@ -30,6 +32,9 @@ The **org.freedesktop.ratbag1.Device** interface provides:
 - Methods:
   - GetProfileByIndex(uint) -> returns the object path for the given index
   - Commit() -> commit the changes to the device
+  - GetSvg(string) -> returns the svg for the theme given, must be one of
```

Comment:
> Also, since `ratbagd_device_get_theme_svg` can return an empty string (through `goto out`), maybe you should mention this as well.

Well, we should probably mention that there is no guarantees that the given path actually exists. `ratbagd` doesn't check if there is a file at this path, so the caller needs anyway to check the returned value (and fall back to something else).